MC/config/PWGHF/pythia8/decayer/force_hadronic_B.cfg

```diff
@@ -0,0 +1,18 @@
+### Force golden B-hadron hadronic decay modes
+### Based on AliRoot AliDecayerPythia8::ForceBeautyUpgrade, latest commit 3154cf8 on Nov 30, 2020
+
+# Bs -> Ds- pi+
+531:onMode = off
+531:onIfMatch = 431 211
+# Lb: 50% to Lc any, 50% to Lc pion
```

Comment:
We will have to figure out a way to replicate in O2DPG the random extraction that was available in AliRoot, as here https://github.com/alisw/AliRoot/blob/master/PYTHIA8/AliPythia8/AliDecayerPythia8.cxx#L686 (unless there is already a better way to do it)

Reply:
this can be achieved by adjusting the branching fractions of the decay channels, I guess